### DISCUSSION TITLE
Add support for bilibili.com

### DISCUSSION THREAD

**Discussion by supaethan** [subhra74/xdm#999] [Ideas]
Hi,
bilibili.com is a very popular site. Would you please make media grabber work on the site? I'm currently using 8.0.18 BETA, media grabber can't pick up any videos on the site.

**Comment by makkash2** [comment]
also would you please show how to create for [kobony.com](https://kobony.com/)

**Reply by linpoui** [reply]
额

**Comment by ghost** [comment]
bilibili is very popular and is the youtube of China. this will help a lot of users.
